public/static/Home.html

```diff
@@ -1,44 +1,50 @@
+<!DOCTYPE html>
+<html>
+
 <head>
 <title>marleigh - Home</title>
 </head>
 
 <body>
-<h1>HOME</h1>
-<br>
-<br>
-<br>
-<nav>
+  <div class = "whole-page">
+<h1 class = "main">HOME</h1>
   <nav>
   <a href="about_me">Me</a> |
-  <a href="current_projects">Current Projects </a> |
+  <a href="projects">Current Projects </a> |
   <a href="blog">Blog</a> |
-</nav>
-</nav>
+  <a href="espresso">Espresso</a> |
 
-<br>
-<br>
+  </nav>
 
-<img src= "images/charleston.jpg" alt="Charleston, South Carolina sunset " align="middle"> 
 
-<br>
-<br>
+<div class = "proper">
+<img src= "images/charleston.jpg" alt="Charleston, South Carolina sunset " align="middle"> 
+</div>
 
 <p>Where can ya find me?</p>
-<br>
-<br>
 
+<p>
 <a href= "https://twitter.com/camarleigh" target = "_blank">twitter</a>
-<br>
+</p>
 
+<p>
 <a href= "https://instagram.com/camarleigh/" target = "_blank">instagram</a>
-<br>
+</p>
 
+<p>
 <a href = "https://github.com/camarleigh" target = "_blank"> github </a>
-<br>
+</p>
```

Comment:
Great set of changes! It took me years to unlearn the habit of using `<br>`s instead of closing my `<p>`s :smile: